packages/x-components/src/x-installer/api/api.types.ts

```diff
@@ -82,6 +82,8 @@ export interface SnippetConfig {
   currency?: string;
   /** Callbacks to be triggered when an XEvent is emitted. */
   callbacks?: XEventListeners;
+  /** Flag determining if the page is a single page application or not. */
+  isSPA?: boolean;
```

Comment:
I prefer this syntax, but the :lidl: does not, rename it to `isSpa`, which is a much funnier name.

Reply:
You're right. I totally forgot to change it.